app/src/app/components/stix/list-property/list-property.component.ts

```diff
@@ -29,6 +29,10 @@ export interface ListPropertyConfig {
     object: StixObject | [StixObject, StixObject];
     /* Edit mode. Default: 'any' */
     editType?: "select" | "stixList" | "any";
+    /* If true, the field will be disabled. Default false if omitted. */
+    disabled?: boolean;
+    /* If true, the field will be required. Default false if omitted. */
+    required?: boolean;
```

Comment:
This doesn't seems to appear correctly in the UI when the edit type is "any":
![Screen Shot 2021-10-04 at 11 30 30 AM](https://user-images.githubusercontent.com/28868888/135880062-0fc38960-3852-472c-a779-6ea6f5d8a808.png)
The asterisk is present, but the field should be red. The red field color is set [here](https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/blob/develop/app/src/style/theme.scss#L158-L160) since normally in Angular Material the red only shows up when the field is both dirty (edited) AND invalid, and we wanted to remove the requirement that it is dirty for the red color to show up.